server/autolinkplugin/config.go

```diff
@@ -148,6 +160,21 @@ func getAutoCompleteData() *model.AutocompleteData {
 	help := model.NewAutocompleteData("help", "", "Autolink plugin slash command help")
 	autolink.AddCommand(help)
 
+	experimental := model.NewAutocompleteData("experimental", "", "Enable experimental feature")
+	autolink.AddCommand(experimental)
+
+	app := model.NewAutocompleteData("app", "", "Enhance UX by making us of the Apps Framework")
+	value := []model.AutocompleteListItem{{
+		HelpText: "Enable",
+		Item:     "on",
+	}, {
```

Comment:
Do we need an off option?

Reply:
Yes, I think so for the same reason as in https://github.com/mattermost/mattermost-plugin-autolink/pull/164#discussion_r671522840.